Kitodo-IdentifierGenerator/src/test/java/org/kitodo/identifiergenerator/DNBURNIdentifierGeneratorTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.identifiergenerator;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for PersistentIdentifier.
+ */
+public class DNBURNIdentifierGeneratorTest {
+
+    @Test
+    public void shouldGenerateUnifiedResourceName() throws Exception {
```

Comment:
It is possible to add more tests which are proving that this used algorithm is correct? At least different values for all four parameters should be used.